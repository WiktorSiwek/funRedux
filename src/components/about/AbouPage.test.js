import AboutPage from './AboutPage';
import { shallow } from 'enzyme';
import React from 'react';

describe('AboutPage', () => {
    const about = shallow(<AboutPage />)
    it('checks page loading', () => {
        expect(about).toMatchSnapshot();
    })
})
