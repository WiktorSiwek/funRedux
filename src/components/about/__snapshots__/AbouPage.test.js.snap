// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage checks page loading 1`] = `
ShallowWrapper {
  "length": 1,
  "node": <div>
    <h1>
      About
    </h1>
    <p>
      This applications uses React, Redux, React Router and a variety of other helpful libraries.
    </p>
  </div>,
  "nodes": Array [
    <div>
      <h1>
        About
      </h1>
      <p>
        This applications uses React, Redux, React Router and a variety of other helpful libraries.
      </p>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_context": Object {},
      "_currentElement": <AboutPage />,
      "_instance": AboutPage {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_nativeContainerInfo": null,
      "_nativeParent": null,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": Object {
        "_currentElement": <div>
          <h1>
            About
          </h1>
          <p>
            This applications uses React, Redux, React Router and a variety of other helpful libraries.
          </p>
        </div>,
        "_renderedOutput": <div>
          <h1>
            About
          </h1>
          <p>
            This applications uses React, Redux, React Router and a variety of other helpful libraries.
          </p>
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": null,
      "_topLevelWrapper": null,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <AboutPage />,
}
`;
